db/migrate/20180718045627_acts_as_taggable_on_migration.acts_as_taggable_on_engine.rb

```diff
@@ -0,0 +1,36 @@
+# This migration comes from acts_as_taggable_on_engine (originally 1)
+if ActiveRecord.gem_version >= Gem::Version.new('5.0')
+  class ActsAsTaggableOnMigration < ActiveRecord::Migration[4.2]; end
+else
+  class ActsAsTaggableOnMigration < ActiveRecord::Migration; end
+end
+ActsAsTaggableOnMigration.class_eval do
```

Comment:
Can you get rid of the old one? The following should be enough :)
```ruby
class ActsAsTaggableOnMigration < ActiveRecord::Migration[4.2]
  def self.up
    # etc
```

Reply:
hey @lucascaton , these migrations were was generated by the gem, so I didnt want to change the original